packages/react-ui-core/src/Ratings/RatingBar.js

```diff
@@ -41,10 +43,10 @@ export default class RatingBar extends PureComponent {
           classnames(
             theme.RatingBar_Item,
             theme[`RatingBar_Item-${index}`],
-            color && theme[`RatingBar_Item-${color}`]
+            this.props.color && theme[`RatingBar_Item-${this.props.color}`]
```

Comment:
Why did this need to be changed?

Reply:
color was being pulled out of props, so it wasn't being passed down to the `RatingItem`

Reply:
Ah, ok, makes sense 👍